src/components/atoms/OakP/OakP.tsx

```diff
@@ -22,11 +22,4 @@ export const OakP = styled.p<OakPProps>`
   ${typographyStyle}
   ${colorStyle}
   ${marginStyle}
-
-  a {
-    color: ${(props) =>
-      props.theme &&
-      props.theme.uiColors &&
-      props.theme.uiColors["text-link-active"]};
-  }
```

Comment:
This was completely wrong. `uiColors` is a map of roles to theme colours. In this case the colour was being set to the literal `navy` which just so happens to be a valid CSS colour name!

The pattern of applying style like this to children breaks the encapsulation of styles in children — in this case the `color` of a nested `OakLink` was being overridden with `navy`.